webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/inventory/InventoryProcessor.scala

```diff
@@ -173,23 +171,6 @@ class InventoryProcessor(
       }
     }
 
-    /*
```

Comment:
So you changed you mind and now you  delete this function (not used anymore) ? o, to me!

Reply:
it was making a warning at each compile which drives me mad